Solr integration test is failing

Solr integration test:
`python -m unittest solr/test_solr.py`
is failing with an error:
> AssertionError: Failed to create core or collection. Error: -bash: /opt/solr/bin/solr: No such file or directory
After switching installation method to bigtop package solr_dir has changed.

Once changed to correct path `/usr/lib/solr`, then it turned out that tests in HA clusters mode fails with an error:

> ERROR: Failed to create collection 'films' due to: {test-solr-ha-1-3-20190430122558-q245cjk8-m-1:8983_solr=org.apache.solr.client.solrj.impl.HttpSolrClient$RemoteSolrException:Error from server at http://test-solr-ha-1-3-20190430122558-q245cjk8-m-1:8983/solr: Error CREATEing SolrCore 'films_shard1_replica_n1': Unable to create core [films_shard1_replica_n1] Caused by: test-solr-ha-1-3-20190430122558-q245cjk8, test-solr-ha-1-3-20190430122558-q245cjk8-m-2:8983_solr=org.apache.solr.client.solrj.impl.HttpSolrClient$RemoteSolrException:Error from server at http://test-solr-ha-1-3-20190430122558-q245cjk8-m-2:8983/solr: Error CREATEing SolrCore 'films_shard1_replica_n2': Unable to create core [films_shard1_replica_n2] Caused by: test-solr-ha-1-3-20190430122558-q245cjk8, test-solr-ha-1-3-20190430122558-q245cjk8-m-0:8983_solr=org.apache.solr.client.solrj.impl.HttpSolrClient$RemoteSolrException:Error from server at http://test-solr-ha-1-3-20190430122558-q245cjk8-m-0:8983/solr: Error CREATEing SolrCore 'films_shard2_replica_n4': Unable to create core [films_shard2_replica_n4] Caused by: test-solr-ha-1-3-20190430122558-q245cjk8}

Additionally in logs its visible connection error:
`Caused by: java.net.UnknownHostException: solr-ha`

--- a/solr/solr.sh
+++ b/solr/solr.sh
@@ -71,7 +71,7 @@ function install_and_configure_solr() {
 
   # Enable hdfs as default backend storage.
   if [[ "${MASTER_ADDITIONAL}" != "" ]]; then
-    solr_home_dir="hdfs://${CLUSTER_NAME}:8020/solr"
+    solr_home_dir="hdfs://${CLUSTER_NAME}-m-0:8020/solr"
   else
     solr_home_dir="hdfs://${CLUSTER_NAME}-m:8020/solr"
   fi

--- a/solr/verify_solr.py
+++ b/solr/verify_solr.py
@@ -12,7 +12,7 @@ Solr performs indexing on data in collections(cores in SolrStandalone mode). Thi
 import subprocess
 import json
 
-SOLR_DIR = '/opt/solr'
+SOLR_DIR = '/usr/lib/solr'
 SOLR_EXAMPLE_DOC = 'https://raw.githubusercontent.com/apache/lucene-solr/master/solr/example/films/films.json'
 SOLR_COLLECTION_NAME = 'films'
 

